public/js/silder.js

```diff
@@ -0,0 +1,39 @@
+var slideIndex = 1;
+showSlides(slideIndex);
+var prev = document.querySelector('.prev');
+var next = document.querySelector('.next');
+var dot = document.querySelector('.dot');
+// dot 
+
+prev.addEventListener('click', ()=>{
+    showSlides((slideIndex += -1));  
+})
+
+next.addEventListener('click', ()=>{
+    showSlides((slideIndex += -1));  
+})
+
+
+function currentSlide(n) {

```

Comment:
Why are you using different format of functions? at line 8 and 12 there are arrow functions used, be consistent with coding style.